### PR TITLE
Course view

### PR DESCRIPTION
**Pull request by sebass482**
# What's here (& why)

-[x] Styling changes
-[x] New font (can be changed)
-[x] Validation changed to have no limit
-[x] NavBar and Logo changed 
-[ ] Modules rendering properly 

# Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete / work-in-progress, PR is for discussion or feedback
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How has this been tested?

- [x] Tested locally

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] There are no merge conflicts

### CODE REVIEW

**Review by BUGBOUNTYchrisg8691 — APPROVED** [review]
looks good

File: src/components/Main/AdminLanding.js [hunk 1]
```diff
@@ -82,6 +82,7 @@ const AdminLanding = props => {
         visible={programModal.visible}
         onCancel={handleCloseModal}
         onFinish={handleCloseModal}
+        closable="true"
```
Comment (chazkiker2) [inline src/components/Main/AdminLanding.js]: the `props.closable` actually defaults to true with Ant Design Modals (so this line can be deleted).

Also worth noting —— `"true"` and `true` are different values in both JavaScript and JSX. Adding `closable="true"` is not the same as saying `closable={true}`. That said, the string value `"true"` is in fact _truthy_ in JS, so `closable="true"` would trickle down to a boolean value of `true`... but that would happen ONLY in JavaScript (and _potentially_ TypeScript and Python... depending on how they treat JSX). Most languages would not be cool with using `"true"` in place of  `{true}`.

File: src/components/Main/Main.js [hunk 2]
```diff
@@ -48,12 +49,18 @@ const Main = props => {
 
   return (
     <Styled.Content>
+      <style>
```
Comment (chazkiker2) [inline src/components/Main/Main.js:52]: yooo I didn't even know you could toss this into a component like this, that's kinda sick

File: src/components/RestCourses/EditCourseForm.js [hunk 3]
```diff
@@ -5,7 +5,9 @@ import '../../styles/Form.css';
 import 'antd/dist/antd.css';
 import { Button, Input, Select, Form } from 'antd';
 import { client } from '../../utils/api';
+import { useHistory } from 'react-router-dom';
 const { TextArea } = Input;
+// let {push} = useHistory()
```
Comment (chazkiker2) [inline src/components/RestCourses/EditCourseForm.js]: we can delete commented-out code

File: src/components/RestCourses/EditCourseForm.js [hunk 4]
```diff
@@ -45,6 +45,7 @@ function EditCourseForm(props) {
     };
 
     client.patchCourse(editedCourse.courseid, editedCourse);
+    // push('/main')
```
Comment (chazkiker2) [inline src/components/RestCourses/EditCourseForm.js]: we can delete commented-out code

**Review by chazkiker2 — CHANGES_REQUESTED** [review]
This is a super clean PR, my friend. Just delete the two instances of commented-out code and maybe get rid of `closeable="true"` and I'll promptly switch this to an approval.

**Review by chazkiker2 — APPROVED** [review]
Went ahead and delete the two lines of commented-out code and the default value passed in to the modal.

**Pull request merged**